sklearn/utils/tests/test_multiclass.py

```diff
@@ -89,7 +100,7 @@ def _generate_sparse(
         np.array([[1, 0, 2, 2], [1, 4, 2, 4]], dtype=np.float32),
         *_generate_sparse(
             [[1, 0, 2, 2], [1, 4, 2, 4]],
-            matrix_types=(csr_matrix, csc_matrix),
+            sparse_containers=[*CSC_CONTAINERS, *CSR_CONTAINERS],
```

Comment:
```suggestion
            sparse_containers=CSC_CONTAINERS + CSR_CONTAINERS,
```

Reply:
I'm going to do `tuple(xxx + xxx)` as suggested in https://github.com/scikit-learn/scikit-learn/pull/27274#discussion_r1321884824 for now. Please let me know if this is not necessary.